### ISSUE TITLE
Read input time series in several time intervals

### ISSUE THREAD

**Issue by sivogel** [hydpy-dev/hydpy#61]
Currently it is not possible to read one input series for one element from several input files.
Reason is that in the method adjust_short_series all values which are within the general timegrid but not within the timegrid of the input file are set to nan or the default value respectively. A new option could allow that the values not within the timegrid are not set to the default, but to the old series. This could look like:

```python
        if hydpy.pub.options.resetseries == False:
            values = self.series
        else:
            values = numpy.full(self.seriesshape, float(self.initinfo[0]))
```

**Comment by sivogel** [comment]
This issue is not solved yet, because input data still cannot but read from different files. But in order to handle long time series (which would not fit into memory) the "just in time" reading and writing meachanism in netcdf mode helps.

**Comment by tyralla** [comment]
I implemented it as suggested in cbcb60e130e144463ab2af10da2094ec855c22b7, but added the option to class `SequenceManager` so I could shorten its name to `reset`.
